pkgs/development/python-modules/magic-wormhole/default.nix

```diff
@@ -3,7 +3,6 @@
 , fetchPypi
 , pythonAtLeast
 , nettools
-, glibcLocales
```

Comment:
Could you edit the second commit message and explain why you removed this?

Reply:
Tests work without it, so why should it be included?

Reply:
Did you try to run the binaries? Asking because the checkbox isn't ticked. :+1:

Reply:
No I didn't since I don't use magic-wormhole. I just came across the build failure.
However, I only ever saw `glibcLocales` added to make the tests run by setting something like `LC_ALL`.